### ISSUE TITLE
Pain disproportionate to abnormality on physical exam

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10695]
**Preferred term label:**
Pain disproportionate to abnormality on physical exam

**Synonyms**

**Definition (free text, please give PubMed ID)**
Exquisite tenderness in a specific region, beyond what one would surmise from examination

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Orthopedists consider this finding to be highly alerting for several disease:

Necrotizing fasciitis
Gas gangrene

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001123
- label: Pain disproportionate to abnormality on physical exam
- parents: [Pain characteristic (HP:0025280)](https://hpo.jax.org/app/browse/term/HP:0025280)
- pmid: 
- def: Exquisite tenderness in a specific region, beyond what one would surmise from examination.
- comment: This finding may be observed with several diseases including necrotizing fasciitis and gas gangrene.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10695
- synonyms:
